### PR TITLE
chore: bump version to v7.3.0

### PR DESCRIPTION
**Pull request by howar31**
# Issue
go-api v7.3.0 release for 支持方案

# Notice

# Dependency
[cloudsql/membership/update_roles](https://github.com/twreporter/migration-history/tree/master/cloudsql/membership/update_roles)

### CODE REVIEW

**Comment by nickhsine** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -254,30 +133,10 @@
...
-
-### Notable Changes
-
-- bump version to trigger CI/CD for applying new k8s config
-
-### Commits
-
-## Released
-
 ### 7.2.3 (Current)
```
Comment (Aylie-Chou) [inline CHANGELOG.md:140]: Could we also update `Current` note to new version

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,118 +1,84 @@
...
-  - add onboarding route & set cookie after onboarding
-
-### Commits
-
-- [[`e50029497b`](https://github.com/nodejs/node/commit/e50029497b)] - **fix**: add user id verification for `onboarding` route (Aylie Chou)
-- [[`2c9a797d28`](https://github.com/nodejs/node/commit/2c9a797d28)] - **fix**: add onboarding route & set cookie after onboarding (Aylie Chou)
+## Released
 
-### 7.2.4-rc.14
+### 7.2.4
```
Comment (Aylie-Chou) [inline CHANGELOG.md:5]: Question: why do we increase the patch number not minor number in this version ?
IMO, patch number increases when bug fix & minor number increase when new feature added.

Comment (howar31) [reply CHANGELOG.md:5]: changed to `7.3.0`

File: CHANGELOG.md (hunk 1 above)
Comment (howar31) [reply CHANGELOG.md]: done

**Pull request merged**